VaderSharp/VaderSharp/SentiText.cs

```diff
@@ -27,20 +28,20 @@ public SentiText(string text)
         private Dictionary<string, string> WordsPlusPunc()
         {
             string noPuncText = Text.RemovePunctuation();
-            var wordsOnly = noPuncText.Split().Where(x=>x.Length > 1);
+            var wordsOnly = Regex.Matches(noPuncText, "\\S{2,}");
```

Comment:
@Chr1st1anSzech Similarly here, just ran some performance tests and it's significantly slower to use RegExs. There may be particular cases where regex is faster - if you can find them, then post the code and let me know. 

Otherwise, I'd rather leave the code as it is (even if unorthodox) - because some applications, including my own, need to do this with live streaming data and any performance improvements however small are important.